xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -122,19 +122,18 @@ static void fetchDisplayModes()
 
 namespace
 {
-const std::map<int, std::string> hdrTypeMap = {
-    {CAndroidUtils::HDRTypes::HDR10, "HDR10"},
-    {CAndroidUtils::HDRTypes::HLG, "HLG"},
-    {CAndroidUtils::HDRTypes::HDR10_PLUS, "HDR10+"},
-    {CAndroidUtils::HDRTypes::DOLBY_VISION, "Dolby Vision"}};
-
 std::string HdrTypeString(int type)
 {
-  auto hdr = hdrTypeMap.find(type);
-  if (hdr != hdrTypeMap.end())
-    return hdr->second;
-
-  return "Unknown";
+  if (type == CJNIDisplayHdrCapabilities::HDR_TYPE_HDR10)
+    return "HDR10";
+  else if (type == CJNIDisplayHdrCapabilities::HDR_TYPE_HLG)
+    return "HLG";
+  else if (type == CJNIDisplayHdrCapabilities::HDR_TYPE_HDR10_PLUS)
+    return "HDR10+";
+  else if (type == CJNIDisplayHdrCapabilities::HDR_TYPE_DOLBY_VISION)
+    return "Dolby Vision";
+  else
+    return "Unknown";
```

Comment:
I know this is mostly personal preference only, but did you replace modern easy extendable code using std::map with old-school if-else-if-else-if-...-else spaghetti code?

Reply:
It seemed to me that the `find` function did not operate correctly after the change, I preferred to solve it in a simple way using several `if...elses` and I think it's fine :)

I also tried using a `case` but it doesn't work with non-constant expressions.

Reply:
Up to you.

Reply:
I left it as it was using `std::map`